src/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation/build/netstandard2.0/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.targets

```diff
@@ -61,6 +60,22 @@
   </Target>
 
   <Target
+    Name="_RunForCoreWithRID"
+    DependsOnTargets="_ResolveInputArguments;_CreateResponseFileForMvcRazorPrecompile"
+    Condition="'$(TargetFrameworkIdentifier)'=='.NETCoreApp' AND '$(RuntimeIdentifier)'!=''">
+
+    <ItemGroup>
+      <_RazorCompilationProject Include="$(MSBuildProjectFullPath)">
+        <AdditionalProperties>RuntimeIdentifier=;MvcRazorOutputPath=$(MvcRazorOutputPath)</AdditionalProperties>
+      </_RazorCompilationProject>
+    </ItemGroup>
+
+    <MSBuild
+      Projects="@(_RazorCompilationProject)"
+      Targets="Build;MvcRazorPrecompile" />
```

Comment:
So this builds the users project without a RID?

Reply:
Yeah, that was the planned fix: https://github.com/aspnet/MvcPrecompilation/issues/102#issuecomment-306591615